plugins/com.google.cloud.tools.eclipse.appengine.localserver/src/com/google/cloud/tools/eclipse/appengine/localserver/server/LocalAppEngineServerBehaviour.java

```diff
@@ -163,11 +178,11 @@ void startDevServer(List<File> runnables, MessageConsoleStream console, String h
    *
    * @param runnables the path to directories that contain configuration files like appengine-web.xml
    * @param console the stream (Eclipse console) to send development server process output to
-   * @param host the host name to which application modules should bind
+   * @param port the app service port
```

Comment:
There is some complication involved here. If we determine the port in `Behavior`, there should be a way to pass the actual port used to `LaunchConfigurationDelegate` so that `openBrowserPage` can know which port to use. What could be the way? FYI, you can't get it from `IServer`, since there is no way you can update server attributes inside `Behavior`. (`IServer` is basically immutable.)